### ISSUE TITLE
Reading a Custom dataset other than olivetti

### ISSUE THREAD

**Issue by sameerg07** [paschalidoud/feature-aggregation#1]
I have tried with OLR dataset which you have written , it works perfectly fine.
I wanted to now experiment on other dataset like grimace , i ordered the this grimce same in the way like OLR, but I am not getting any results . Many errors. So can you please write up a python file to do that .

**Comment by angeloskath** [comment]
I am not sure I understand your issue. Are you having problems with the feature aggregation or with reading the dataset?

**Comment by sameerg07** [comment]
Yeah , @angeloskath  i had doubts on how you have resized the images, in the sense , when i tried doing with the grimace data, it is overfitting and giving very high results. Should we compulsarily graayscale the i images? I wanted to know what exactly the 100 in Vlad(100) mean? like does it vary with the input size and stuff
